qlib/rl/order_execution/reward.py

```diff
@@ -21,10 +21,13 @@ class PAPenaltyReward(Reward[SAOEState]):
     ----------
     penalty
         The penalty for large volume in a short time.
+    zoom
```

Comment:
Suggest "scale"

Reply:
@lihuoran  Why is `zoom`/ `scale` necessary if we have hyperparameters like learning rate?

Reply:
Will be renamed to "scale".
@you-n-g In the AAAI 2021's open-source project, they use a scaled reward. I personally think adding this parameter would be more flexible.